IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CableSegmentTypeFiberSegment/DocPropertySet.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CableSegmentTypeFiberSegment" UniqueId="41f6e212-3454-43fd-229a-68e45fd425f7" ApplicableType="IfcCableSegment/FIBERSEGMENT" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
Is this an instance property set?  Is Assigned and function don't seem like type properties.

Reply:
Properties has been moved to cablesegmentoccurencefibersegment
Commit : 4d5e83c1bf208182fa5e5d0b2648e61d35e95f5a